tools/capi_test/main.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright(c) Microsoft Corporation.All rights reserved.
```

Comment:
When possible, I'd prefer to not maintain such code in this repo. In the past we had similar code in the csharp folder and it soon became broken. Because it is not covered by CI build and when something was went wrong we need to go through a long progress to fix it. Especially when the packaging pipeline is already broken, it would take much longer to be noticed.

Reply:
ADO pipelines can support multiple repos: https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops

Reply:
How about moving this simple code to onnxruntime-inference-examples?